pkgs/applications/science/machine-learning/weights/keras/resnet50.nix

```diff
@@ -0,0 +1,12 @@
+{ stdenvNoCC, fetchurl, top ? false }:
+
+import ./common.nix {
+  inherit stdenvNoCC fetchurl;
+  model = "resnet50";
+  version = "2018-08-06";
+  weights =
+    if top
+    then "resnet50_weights_tf_dim_ordering_tf_kernels.h5"
+    else "resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5";
+  sha256 = "017m98glf5jm9qwsb64mki6s28by2qm3s3pkqidw3z7kmwyv9j36";
```

Comment:
You can't have the same hash for two different files.
It would be easier to just ship both files for the `resnet50` variant.
Then it would up to the consumer (app or user) of the package to use either `${keras-resnet50}/resnet50_weights_tf_dim_ordering_tf_kernels.h5` or `${keras-resnet50}/resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5

Reply:
I'm thinking separate derivations then. But if I were to do this, how would `src = ... ` work? I keep forgetting how multiple srcs work for derivations.